shared/src/main/scala/amf/core/client/common/remote/Content.scala

```diff
@@ -4,15 +4,28 @@ import amf.core.client.scala.lexer.{CharSequenceStream, CharStream}
 
 import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
 
-case class Content(stream: CharStream, url: String, mime: Option[String] = None) {
+case class Content(stream: CharStream, url: Option[String], mime: Option[String] = None) {
```

Comment:
Could we make these variables private and leave accessors for them ? I'd like to hide the "stream" class implementation